tests/app/parser/test_metadata_parser.py

```diff
@@ -308,6 +308,26 @@ def test_invalid_schema_metadata(self):
 
         self.assertEqual('Invalid validator for schema metadata - invalidValidator', ite.exception.args[0])
 
+    def test_optional_metadata(self):
+        metadata = {
+            'jti': str(uuid.uuid4()),
+            'user_id': '1',
+            'form_type': 'a',
+            'collection_exercise_sid': 'test-sid',
+            'eq_id': '2',
+            'period_id': '3',
+            'ru_ref': '2016-04-04',
+            'period_str': 'May 2016',
+            'case_id': str(uuid.uuid4())
+        }
+
+        self.schema_metadata.append({'name': 'address_line1', 'validator': 'string', 'optional': True})
+        validate_metadata(metadata, self.schema_metadata)
+
+        metadata['address_line1'] = '123 awesome place'
+        validate_metadata(metadata, self.schema_metadata)
```

Comment:
You're not actually asserting anything in this whole test.  Probably want to actually test something :P

Reply:
I looked at the file again, theres another test that does it in the same way.  Feel free to leave it as is, but I think validate_metadata returns a True/False, might be a simple as adding assertTrue/False to it

Reply:
It not raising an exception is the test.
You could just add an optional metadata into `self.schema_metadata`. Then it would be tested as part of:
https://github.com/ONSdigital/eq-survey-runner/blob/496d70d4cc9e1c9fc3f9348098edd10486c3289e/tests/app/parser/test_metadata_parser.py#L94-L96

Reply:
Good shout 👍